atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -0,0 +1,60 @@
+from functools import wraps
+
+import atlassian_jwt_auth
+from django.conf import settings
+from django.http.response import HttpResponse
+from requests.exceptions import (HTTPError, ConnectionError)
+from jwt.exceptions import (InvalidIssuerError, InvalidTokenError)
+
+from atlassian_jwt_auth.contrib.django.utils import parse_jwt, verify_issuers
+
+
+def requires_asap(issuers):
+    """Decorator for Django endpoints to require ASAP
+
+    :param list issuers: (required) The 'iss' claims that this endpoint is from
+    """
+    def requires_asap_decorator(func):
+        @wraps(func)
+        def requires_asap_wrapper(request, *args, **kwargs):
+            verifier = _get_verifier()
```

Comment:
I haven't checked how if creating a new `verifier` like this has an impact on cachecontrol (and caching of public keys) but you may want to check.